docker-testing/Dockerfile-fetchers

```
@@ -0,0 +1,17 @@
+FROM netmeld-test-base
+
+ENV NETMELD_DB netmeld_test_fetchers
+
+WORKDIR /home/netmeld/setup
+
+RUN apt install --quiet --quiet --assume-yes --option Dpkg::Use-Pty=0 \
+      --no-install-recommends --no-install-suggests \
+      ./netmeld-*core.deb ./netmeld-*fetchers.deb ./netmeld-*tool-clw.deb
```

Couple of things with this,

- While the `--no-install-suggests` might be ok, we want to install recommends. The use cases for not installing them are special corner cases we don't need to test under this Issue/PR.
- We don't want to explicitly call out dependencies that should already be installed prior to building this container.
- There should be a different Dockerfile for each module.

```suggestion
RUN apt install --quiet --quiet --assume-yes --option Dpkg::Use-Pty=0 \
      --no-install-suggests \
      ./netmeld-*fetchers.deb
```